libecho/tests/tmp_test.cpp

```diff
@@ -1,4 +1,4 @@
-#include "Echo.h"
```

Comment:
Instead of commenting out most of the tests you could just leave that include and change only the test you've changed.

Reply:
Those tests will be deleted anyway - you can just do it now.